### PR TITLE
[home] Pop to artist tab when coming from background w/selected artist

### PR DESCRIPTION
**Pull request by sarahscott**
This makes sure that Home shows the Artists tab when there's a selected artist even when it's coming from the background and was originally on a different tab.

Fixes https://github.com/artsy/collector-experience/issues/751

Skip New Tests

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  [danger-plugin-jest] Could not read test results. Danger cannot pass or fail the build.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Scenes/Home/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+"use strict";
```
Comment (mzikherman) [inline src/lib/Scenes/Home/index.js]: I think this may have been committed by accident?

Comment (sarahscott) [reply src/lib/Scenes/Home/index.js]: Interesting! I just removed it - let's see if that fixes the CI.

File: src/lib/Scenes/Home/index.tsx [hunk 2]
```diff
@@ -17,26 +17,54 @@ import renderWithLoadProgress from "lib/utils/renderWithLoadProgress"
...
+const TabBarContainer = styled.View`margin-top: 20px;`
 
 interface Props {
   selectedArtist?: string
   selectedTab?: number
 }
 
-export default class Home extends React.Component<Props, null> {
+interface State {
+  appState?: string
```
Comment (mzikherman) [inline src/lib/Scenes/Home/index.tsx]: Is this actually optional?

**Comment by mzikherman** [comment]
I just pushed a commit, since this was strangely still ❌ on Circle. Seemed like a file should have been updated in https://github.com/artsy/emission/pull/899 and was causing a weird error while running `yarn ci`.

Will merge on 💚

File: src/lib/Scenes/Home/index.tsx [hunk 3]
```diff
@@ -26,12 +26,43 @@ interface Props {
   selectedTab?: number
 }
 
-export default class Home extends React.Component<Props, null> {
+interface State {
+  appState: string
+}
+
+export default class Home extends React.Component<Props, State> {
+  tabView?: ScrollableTabView | any
```
Comment (mzikherman) [inline src/lib/Scenes/Home/index.tsx]: This typing `| any` also seems weird to me...but I don't know enough to really say more. 👻

**Pull request merged**